src/messaging/ExchangeContext.h

```diff
@@ -177,7 +177,6 @@ class DLL_EXPORT ExchangeContext : public ReliableMessageContext,
 
     // Applies a suggested response timeout value based on the session type and the given upper layer processing time for
     // the next message to the exchange. The exchange context must have a valid session when calling this function.
-    //
```

Comment:
Why this change?  Please don't make this change.

Reply:
done, thanks